mobile/src/main/java/ch/epfl/sdp/mobile/ui/social/SocialScreenState.kt

```diff
@@ -0,0 +1,35 @@
+package ch.epfl.sdp.mobile.ui.social
+
+import androidx.compose.runtime.Stable
+
+/**
+ * A state which indicates the content of an [SocialScreen] composable. It will keep track of the
+ * values of research text field.
+ *
+ * This state contains a [Mode] that describe which screen to display
+ */
+@Stable
+interface SocialScreenState {
+
+  /** Indicate if displayed screen is the Following or the Searching screen */
+  enum class Mode {
+
+    /** The user can scroll a list containing the followed player */
+    Following,
+
+    /** The user is searching in the search bar */
+    Searching,
+  }
+
+  /** The current [Mode] */
+  var mode: Mode
+
+  /** The [List] of players that need to be displayed */
+  var players: List<Person>
+
+  /** The current user input in the searc bar */
```

Comment:
Typo :

```suggestion
  /** The current user input in the search bar */
```

Reply:
Fixed in the commit https://github.com/epfl-SDP/android/pull/91/commits/43f7ba7d10076a2c485150ea3434fab325daa8d0